packages/db/src/provider.ts

```diff
@@ -0,0 +1,59 @@
+import omit from 'lodash/omit';
+import isNil from 'lodash/isNil';
+import isValidHostname from 'is-valid-hostname';
+
+import { ProviderRecord, PostgresProviderRecord } from '@cumulus/types';
+import { PostgresValidationError } from '@cumulus/errors';
+
+import KMS from '@cumulus/aws-client/KMS';
+
+export const encryptValueWithKMS = (
+  value: string
+): Promise<string> => {
+  if (process.env?.provider_kms_key_id === undefined) {
+    throw new Error('env variable provider_kms_key_id must be set');
+  }
+  return KMS.encrypt(process.env?.provider_kms_key_id, value);
```

Comment:
We shouldn't have to conditionally check for this value on process.env if we're throwing an error above. But the helper from envUtils would take care of it as well